src/gameComponents/Ship/ProtoShip.java

```diff
@@ -0,0 +1,21 @@
+package gameComponents.Ship;
+
+import BBDGameLibrary.Geometry2d.BBDPoint;
+
+/**
+ * A class to serve as a precursor to a proper ship class.  Primarily serves as a testing
+ * bed for building new objects and the like.
+ */
+public class ProtoShip {
+    private enum Size {SMALL, MEDIUM, LARGE}
+
+    private BBDPoint currentLocation = new BBDPoint(0,0);
```

Comment:
also is this the center point, or some weird like front-left corner thing

Reply:
Center center

Reply:
heard